### PR TITLE
First pass on reworking the instructions.

### PR DESCRIPTION
**Pull request by beorn7**
This mostly just updates obsolete entries and fixes a few typos and
formatting issues.

Selected few things have been added.

@juliusv

### CODE REVIEW

File: workshop.md [hunk 1]
```diff
@@ -33,11 +39,15 @@ While a Prometheus server which collects only data about itself is not very
 useful in practice, it is a good starting example.
 
 ## Starting Prometheus
-Start Prometheus. By default, Prometheus stores its database in /tmp/metrics
-(flag `-storage.local.path`).
+
+Start Prometheus. By default, Prometheus reads its config from a file
+called `prometheus.yml` in the current working directory, and it
+stores its database in a sub-directory called `data`, again relative
+to the current working directory. Both behaviors can be changed usung
```
Comment (juliusv) [inline workshop.md]: s/usung/using/

File: workshop.md [hunk 2]
```diff
@@ -197,21 +212,22 @@ adjusts the targets. To configure a job with DNS service discovery, add the
...
 
 # The expression language
 
 With more metrics collected by your Prometheus server, it is time to
-familiarize yourself a bit more with the expression language. For comprehensive
-documentation, see http://prometheus.io/docs/querying/basics/. The following is
-meant as an inspiration for how to play with the metrics currently collected by
-your server. Evaluate them in the *Console* and *Graph* tab. For the latter, try
-different time ranges and the *stacked* option.
+familiarize yourself a bit more with the expression language. For
```
Comment (juliusv) [inline workshop.md]: We might need to standardize on a word-wrapping length or we will go mad... my vim by default uses 79 characters per line + newline.

File: workshop.md [hunk 3]
```diff
@@ -522,22 +565,36 @@ workshop participants sending alerts to the same Alertmanager, you might run
...
-http://prometheus.io/docs/practices/alerting/.
+For meaningful alerting, refer to the
+[best practices section about alerting](http://prometheus.io/docs/practices/alerting/).
+
+Create a useful alerting rule for your example application and try it out. Possible tasks:
+* Create an alert on your service being down. Stop the service and
+  check if and when the alert fires.
+* The example service simulates short outages now and then. Create an
+  alert that will detect them.
+* Modify the code to simulate other kind of outages and create alerts
```
Comment (juliusv) [inline workshop.md]: s/other kind/other kinds/

**Comment by juliusv** [comment]
:+1: otherwise

File: workshop.md (hunk 1 above)
Comment (beorn7) [inline workshop.md]: Done.

File: workshop.md (hunk 2 above)
Comment (beorn7) [inline workshop.md]: I can switch to that, too. No problem.

File: workshop.md (hunk 3 above)
Comment (beorn7) [inline workshop.md]: Done.

**Pull request merged**